### DISCUSSION TITLE
Perturb problem

### DISCUSSION THREAD

**Discussion by yunjian-hash** [deepmodeling/dpdata#381] [Q&A]
Dear developers,

When I use perturb funtion, I meet this error. what is the problem and how to solve this? Here are the dpdata.py and error. Thanks!
![54226c124b612f041f07724664d60fe](https://user-images.githubusercontent.com/75162924/204209157-565f9e1f-85cd-4d0e-83ee-8bf15eda316d.png)

from dpdata import System, LabeledSystem, MultiSystems
import numpy as np

ls3=LabeledSystem('./Walker_2', fmt='cp2k/aimd_output')
a3 = np.random.choice(len(ls3),size=100,replace=False)
w3 = list(set(range(1,len(ls3)))-set(a3))[10:-1:6]
rs3=ls3.sub_system(w3)
perturbed_system_1 = rs3.perturb(pert_num=1, cell_pert_fraction=0.1, atom_pert_distance=0, atom_pert_style='normal') 

ms= MultiSystems() 
ms.append(rs3) 
ms.append(perturbed_system_1)

ms.to_deepmd_raw('deepmd')
ms.to_deepmd_npy('deepmd',set_size=5000)

**Comment by wanghan-iapcm** [comment]
Thank you very much for reporting the bug. A quick fix is using `System` instead of `LabeledSystem`
```
ls3=System('./Walker_2', fmt='cp2k/aimd_output')
```

The fix is PRed as #383
